index.d.ts

```diff
@@ -24,6 +24,10 @@ export interface SerializedError {
    * This is available for subsequent serializers to use.
    */
   raw: Error;
+  /**
+   * `cause` is never included in the log output, if you need the `cause`, use `raw.cause`
```

Comment:
```suggestion
   * `cause` is never included in the log output, if you need the `cause`, use {@link raw.cause}
```

I wonder if we could use `@link` here to get some better highlights: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#link